airflow/models/dag.py

```diff
@@ -357,6 +358,9 @@ class DAG(LoggingMixin):
         Can be used as an HTTP link (for example the link to your Slack channel), or a mailto link.
         e.g: {"dag_owner": "https://airflow.apache.org/"}
     :param auto_register: Automatically register this DAG when it is used in a ``with`` block
+    :param fail_stop: Fails currently running tasks when task in DAG fails
+        **Warning**: A fail stop dag can only have tasks with the default trigger rule ("all_success").
+        An exception will be thrown if any task in a fail stop dag has a non default trigger rule.
```

Comment:
How does this render in documentation? I don’t think the newline would be rendered, and the end result would read weird without a period in the previous sentense.

Reply:
That makes sense. I have updated the documentation so that the `param fail_stop` sentence has a period at the end of it.